funnel/assets/js/project.js

```diff
@@ -0,0 +1,142 @@
+import { Utils, TableSearch } from './util';
+import L from "leaflet";
+
+const defaultLatitude = '12.961443';
+
+const defaultLongitude = '77.64435000000003';
```

Comment:
Where does this point, and why is it the default? Defaults can have interesting consequences:

https://splinternews.com/how-an-internet-mapping-glitch-turned-a-random-kansas-f-1793856052
https://gizmodo.com/how-cartographers-for-the-u-s-military-inadvertently-c-1830758394